docs/architecture/adr-009-evidence-module.md

```diff
@@ -0,0 +1,205 @@
+# ADR 009: Evidence Module
+
+## Changelog
+
+- 31-07-2019: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+
+In order to support building highly secure, robust and interoperable blockchain
+applications, it is vital for the Cosmos SDK to expose a mechanism in which arbitrary
+evidence can be submitted, evaluated and verified resulting in some agreed upon
```

Comment:
Should extend beyond equivocation (double-voting) to misbehaviour, including signing when unbonded, signing an incorrect state transition (in the future), etc.

Reply:
Updated.